.ci/run.sh

```diff
@@ -61,7 +61,7 @@ case "${CI_JOB}" in
 		echo "INFO: Running tracing test"
 		sudo -E PATH="$PATH" bash -c "make tracing"
 		;;
-	"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR")
+	"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
```

Comment:
I'm not sure you mean this? I think your new flags want to run something like:
```
echo "INFO: Running Confidential Container Kubernetes tests"
sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-kubernetes"
```
?

Reply:
that makes more sense - would we then also need a change to `Makefile` to add `cc-kubernetes` as a command and point it to the correct tests?

Reply:
Yes, but I'd expect that change to be with your new test case changes (which could be part of this PR?)

Reply:
yes, please, send the new tests alongside this changes so I can have it tested more easily :)

Reply:
ok that makes sense - in that case I'll close this PR as our test changes are on a different branch so it makes more sense to open a PR from there and add in these changes - thanks for the clarification!